### ISSUE TITLE
Port RadioButton.IsChecked to Handler

### ISSUE THREAD

**Issue by hartez** [dotnet/maui#404]
Please read the [Handler Property PR Guidelines](https://github.com/dotnet/maui/wiki/Handler-Property-PR-Guidelines) for instructions on how to create and submit a PR for this issue.

Thanks for your help!

**Comment by JanNepras** [comment]
May I finish this one?

**Comment by hartez** [comment]
@JanNepras This one might have a wait a bit - RadioButton is on hold while we get the templated control stuff in place. Once #482 is merged, if there's still work to be done you'll be first in line :)
